pkgs/tools/networking/udpreplay/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, cmake, libpcap, fetchgit, lib }:
+stdenv.mkDerivation rec {
+  name = "updreplay";
+  version = "1.0.0";
+  buildInputs = [ cmake libpcap ];
```

Comment:
cmake is a nativeBuildInput. To check cross compilation:
`nix-build -A pkgsCross.aarch64-multiplatform.udpreplay`

Reply:
Oh, yes, of course. I have checked that it now builds against the cross-compilation target above. I haven't tested to run it on an aarch64 platform however.